### PR TITLE
Grafana v8.5.15 update

### PR DESCRIPTION
**Pull request by bulkware**
Both branches (base/compare) have been checked out from the same upstream's tag:
https://github.com/grafana/grafana/releases/tag/v8.5.15

The difference is that "iiris-v8.5.15" branch has Iiris specific changes. After this squash & merge we will create a new "iiris-develop" from the resulting branch.

The Iiris specific changes were very straightforward except for one file: "DashNav.tsx". That file on Grafana repository was converted into a functional component and used React.memo to, I assume, improve performance. The commit for that change on Grafana repository is here:
https://github.com/grafana/grafana/commit/c13f4542d8de59e262aeddb61cd07c52b5db2d5e

We could/should do the same thing, but I am unsure if the Iiris specific changes can be made onto a functional component. It would make sense to follow Grafana's changes as much as possible and not diverge too much. I will reserve a meeting for a closer look with @rwdigia and @blindBombus for this.

You can read more about React.memo here:
https://dmitripavlutin.com/use-react-memo-wisely/

You can see Timo's original changes to version 8.4.7 here if you need them:
https://github.com/digiaiiris/grafana/pull/61

### CODE REVIEW

**Comment by bulkware** [comment]
The meeting to check on "DashNav" component is scheduled for tomorrow :)

**Review by blindBombus — APPROVED** [review]

**Review by sapajune — APPROVED** [review]

File: public/app/app.ts [hunk 1]
```diff
@@ -90,6 +90,15 @@ if (process.env.NODE_ENV === 'development') {
 }
 
 export class GrafanaApp {
+  storedLanguage: string;
+
+  constructor() {
+    // Check for Iiris language
+    this.storedLanguage = localStorage.getItem('iiris_language') || 'fi';
+    console.log('this.storedLanguage: ' + this.storedLanguage);
```
Comment (rwdigia) [inline public/app/app.ts:98]: Maybe omit console.log in production? :)

**Review by rwdigia — APPROVED** [review]
In the coming releases maybe we could try to adjust ESLint and TSLint settings instead of having to disable them in files altogether. Like these:

```
/* eslint-disable */
/* tslint:disable */
```

Also the use of "any" makes TypeScript kind of unnecessary, so that's also something we could strive towards, ie. declaring types.

File: public/app/app.ts (hunk 1 above)
Comment (bulkware) [reply public/app/app.ts]: Good point! In commit 6db155846910b1ad703c3b08d014957ced91d959.

**Review by blindBombus — APPROVED** [review]

**Pull request merged**